platforms/php/prereqs/build.sh

```diff
@@ -61,6 +61,28 @@ if [ $PHP_MAJOR == '5' ]; then
 	versionDevReqs='libssl1.0-dev'
 fi
 
+if [ $PHP_MAJOR == '7' ] && [ $PHP_MINOR == '4' ]; then 
```

Comment:
Do you want to clean up the duplicate files?

Reply:
not in this PR .. I kept this PR in a stable state after many builds :P.. so I would prefer to do the cleanup  seperately